views/pulsar/v2/helpers/form.html.twig

```diff
@@ -116,10 +132,14 @@ data-*        | string | Data attributes, eg: `'data-foo': 'bar'`
     {% import _self as form %}
     {% import '@pulsar/pulsar/v2/helpers/elem.html.twig' as elem %}
 
+    {% if options.help is defined and options.help is not empty %}
+        {% set options = options|merge({ 'help_id': 'guid-' ~ random() }) %}
```

Comment:
Super minor, but looks as though the Pulsar convention for strings with spaces is `kebab-case` rather than `snake_case` for macro argument keys, is it worth being consistent?

Reply:
@mikedevelops there's two things in play here, which I think is what caused the inconsistency, but I don't think this was a conscious decision, we just need to decide whether to keep it or change it:

* If an option ends up in the view (so is really a HTML attribute) it's `kebab-case`, like HTML attributes are
* If an option is internal, it's `snake_case`

Reply:
What category would `show-label` fall into? Isn't that technically internal as it's used to derive the visually hidden class on labels? It was seeing that, that made me think `snake_case` wasn't a thing

Reply:
Yep, that's inconsistent ;)

Reply:
I think it'd just be far easier if we stuck with one convention, hyphens most likely as I think the append_type is the main one that appeared differently

Reply:
I-think-kebab-as-the-convention-would-work

Reply:
So change to `help-id` and we'll change the rest in due course